source/getting-started/onboarding.markdown

```diff
@@ -14,6 +14,12 @@ Set your username and password.
 
 Next, you can enter a name for your home and set your location and unit system. Click "DETECT" to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
 
+<div class='note warning'>
```

Comment:
But default nothing is exposed on the internet.
This is better suited for <https://www.home-assistant.io/docs/configuration/remote/> or even <https://www.home-assistant.io/docs/configuration/securing/>

Reply:
I thought about placing it in one of those locations but it seemed better to me to place it in the location where the information would be entered.

I have no issue with the dev team moving the warning to where they see fit but I think it should be somewhere in the docs.